### ISSUE TITLE
1.12.2 Server crash

### ISSUE THREAD

**Issue by vitasjr** [SleepyTrousers/EnderIO#5315]
I am new to this and might have done a beginner misstake.
I am trying to create a server with a couple of mods, one of them is Ender IO. I tried in single player and everything worked. 

When I copy the mods files to the server it starts up but then creates like 100 lines with something like [Server threaed/INFO] [EnderCode Mixins] and I can't scroll to read the full line since the whole computer starts to lag (I don't have a bad computer)

I can see in the server java file that memory use is at 0% free of 250mb and I tried to create a .bat file to run the server with more allocated ram but the server didn't even start when using the .bat file. 

The server works when I remove Ender IO from the server folder

Forge version : 1.12.2-14.23.5.2847

I found another tread about this that said someting about deleting memory_repo folder but it didn't work either.

Crash report. plz help :) 
[crash-2020-01-19_21.21.44-server.txt](https://github.com/SleepyTrousers/EnderIO/files/4083515/crash-2020-01-19_21.21.44-server.txt)

**Comment by HenryLoenwind** [comment]
`Caused by: java.lang.OutOfMemoryError: Java heap space`

250MB? That's about 1/12 of what a modded server needs...

**Comment by vitasjr** [comment]
> `Caused by: java.lang.OutOfMemoryError`
> 
> 250MB? That's about 1/12 of what a modded server needs...

I know but how do I change to like 4gb or more?? I tried with a .bat file that I found on youtube but nopthing happens when i try to run the file

How do i allocate more ram

**Comment by vitasjr** [comment]
I tried running the .bat file with this script. It opens for like 1 second and then closes. I tried adding pause in the end and got this message

-Xms4096M -Xmx6G -jar forge-1.12.2-14.23.5.2847-universal.jar
Invalid initial heap size: -Xms4096M
The specified size exceeds the maximum representable size.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

I have 16gb of memory what am I doing wrong
